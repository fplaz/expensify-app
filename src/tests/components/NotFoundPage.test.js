import React from "react"
import NotFoundPage from "../../components/NotFoundPage"
import { MemoryRouter } from "react-router-dom"
import {render} from '@testing-library/react'

test('Should render NotFoundPage correctly', () => {
    const {asFragment} = render(
        <MemoryRouter>
            <NotFoundPage />
        </MemoryRouter>
    )
    expect(asFragment()).toMatchSnapshot()
})