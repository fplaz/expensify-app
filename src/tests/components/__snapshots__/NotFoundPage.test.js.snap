// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFoundPage correctly 1`] = `
<DocumentFragment>
  <div>
    <p>
      404!
    </p>
    <a
      href="/"
    >
      Go home
    </a>
  </div>
</DocumentFragment>
`;
